src/common/CommonUtils.ts

```diff
@@ -19,6 +20,24 @@ export class CommonUtils {
         return Promise.resolve();
     }
 
+    public static delay(ms: number): Promise<void> {
+        return new Promise((resolve) => setTimeout(resolve, ms));
+    }
+
+    public static resolvePath(inputPath: string): string {
```

Comment:
I'm also a little dubious of the extent of the functionality in here. We're only calling this—and likely to only call this—in scenarios where we know we need to resolve the various locations that could apply to the user's home directory.

IMO we should just retool this function to return the user's home directory. The rest of it seems like a solution in search of a problem.

Reply:
In this repo yes. But in the internal repo we're going to be needing to do the 3 logic here for example for the configure certificates command. I thought it would be useful to have the logic in a function in common utilities so that after breaking up this repo, any repo that's going to need this logic can just consume it from CommonUtils.

What do you think. Should I still go ahead and revert this?

Reply:
I am very likely missing something in the details, but it feels like—at least from the way we're currently consuming it—that we're duplicating/replicating efforts that the consumer is already making.

For example, the typical input that this method is currently receiving is along the lines of `~/.android/avd/myEmu.avd/emu-launch-params.txt`. It seems like the *only* thing we don't know from that input is what `~` is supposed to resolve to.

Now if we envision future scenarios where we're going to pass less literal, more extensible paths into this method, like say:
```typescript
// anotherDerivedPath got set to something like '../Projects/deep/down/../top-config.txt',
// out of our purview.
const fullPath = CommonUtils.resolvePath(path.join('~', anotherDerivedPath))
```
Then it makes sense to me to have the additional functionality.  As it's currently being used, its primary function seems to be dereferencing the proper home path.

Reply:
You know what, you've convinced me. Let's just simplify this method and make it to only resolve the user home path. Anything else can be done by chaining `path.resolve` and `path.normalize` by the caller.